source/testing/testing-controllers.md

```diff
@@ -30,8 +30,8 @@ export default Ember.Controller.extend({
 });
 ```
 
-`setProps` sets a property on the controller and also calls a method. In our 
-generated test, ember-cli already uses the `moduleFor` helper to setup a test 
+The `setProps` action directly sets one property, and calls the method to set the other.
+In our generated test, ember-cli has already used the `moduleFor` helper to set up a test
```

Comment:
I reverted the sentence starting with "In our generated", except change the word "setup" to "set up", because that's just plain wrong. Is that the part you meant to be reverted? It's hard to tell with the way GitHub is showing the diff.